concordium-contracts-common/src/types.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(feature = "std")]
+use crate as concordium_std;
```

Comment:
Why do you need this only when `std` feature is enabled? I would have assumed you need it always.

Reply:
removed it